HACKING.md

```diff
@@ -302,9 +305,11 @@ Place all your changes in a commit called "Refactor to match changes upstream".
 
 A typical pull request for your Kubernetes rebase will contain:
 
-1. One commit for the Godeps bump (`bump(k8s.io/kubernetes):<commit SHA>`).
-2. Zero, one or more cherry-picked commits tagged UPSTREAM.
-3. One commit "Refactor to match changes upstream".
+1. One commit for the Kuberentes Godeps bump (`bump(k8s.io/kubernetes):<commit SHA>`).
```

Comment:
We agreed that it was a good idea to squash in the unit test dependencies into the bump commit right?  That keeps the history smaller.  @liggitt's sugggestion.